providers/userdetail/src/java/org/sakaiproject/user/detail/CandidateDetailProviderImpl.java

```diff
@@ -54,13 +56,13 @@ public class CandidateDetailProviderImpl implements CandidateDetailProvider {
 	private final static String PROP_ENCRYPT_NUMERIC_ID = "encryptInstitutionalNumericID";
 	private final static String PROP_ENCRYPT_CANDIDATE_DETAILS = "encryptCandidateDetails";
 
-	private PreferencesService preferencesService;
-	private ServerConfigurationService serverConfigurationService;
-	private SessionManager sessionManager;
-	private SiteService siteService;
-	private SecurityService secServ;
-	private ToolManager toolManager;
-	private ValueEncryptionUtilities encryptionUtilities;
+	@Setter private PreferencesService preferencesService;
+	@Setter private ServerConfigurationService serverConfigurationService;
+	@Setter private SessionManager sessionManager;
+	@Setter private SiteService siteService;
+	@Setter private SecurityService securityService;
+	@Setter private ToolManager toolManager;
+	@Setter private ValueEncryptionUtilities encryptionUtilities;
```

Comment:
I'd just put @Setter on the class, personally. You may as well.

Reply:
Makes sense, thanks